internal/pkg/plugin/gitlabci/java/template.go

```diff
@@ -0,0 +1,72 @@
+package java
+
+import (
+	"bytes"
+	"html/template"
+)
+
+var gitlab_ci_template = `
+image: docker:stable
+
+stages:
+  - package
+  - docker_build
+  - k8s_deploy
+
+{{if .Package.Enable}}
+mvn_package_job:
+  image: {{.Package.BaseOption.Image}}
+  stage: package
+  tags:
+    - {{.Package.BaseOption.Tags}}
+  script: 
+    {{range .Package.ScriptCommand}}
+    {{.}}
+    {{end}} 
+  artifacts:
+    paths:
+      - target/*.jar
+{{end}}
+
+{{if .Build.Enable}}
+docker_build_job:
+  image: {{.Build.BaseOption.Image}}
+  stage: docker_build
+  tags: 
+    - {{.Build.BaseOption.Tags}}
+  script:
+    {{range .Build.ScriptCommand}}
+    {{.}}
+    {{end}} 
+{{end}}
+
+{{if .Deploy.Enable}}
+k8s_deploy_job:
+  image: 
+    name: {{.Deploy.BaseOption.Image}}
+    entrypoint: [""]
+  stage: k8s_deploy
+  tags: 
+    - {{.Deploy.BaseOption.Tags}}
+  script:
+    {{range .Deploy.ScriptCommand}}
+    {{.}}
+    {{end}} 
+{{end}}
+`
+
+// Render gitlab-ci.yml template with Options
+func renderTmpl(Opts *Options) (string, error) {
+	t, err := template.New("gitlabci-java").Option("missingkey=error").Parse(gitlab_ci_template)
```

Comment:
If the template is provided by the program and is guaranteed to be correct. You can use `template.Must`

```go
t := template.Must(template.New("gitlabci-java").Option("missingkey=error").Parse(gitlab_ci_template))
```